### ISSUE TITLE
Property 'style' does not exist on type 'Element'

### ISSUE THREAD

**Issue by Aster89** [microsoft/TypeScript#47245]
# Bug Report

### 🔎 Search Terms

  - "Property 'style' does not exist on type 'Element'." 
  - javascript

I've found #16920, but the solution seems to be applicable to TypeScript, not JavaScript.

### 🕗 Version & Regression Information

- This is the behavior in every version I tried (I pasted the two lines of the snippet below into [here](https://www.typescriptlang.org/play?ts=4.5.4#code/MYewdgzgLgBAHjAvDAJiYBXAtgUzFAOgEcMcAnATwGUcAbHYKEMgCgHICB3ACwEMocAN3JsAlAG4AUKEiwKSeAWgV64oA), and error is the same for whichever version I select), and I reviewed the FAQ for entries about "style" and "Element"

### ⏯ Playground Link

[Playground link with relevant code](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAHjAvDAJiYBXAtgUzFAOgEcMcAnATwGUcAbHYKEMgCgHICB3ACwEMocAN3JsAlAG4AUKEiwKSeAWgV64oA)

### 💻 Code

```javascript
// this is some-file.js
const x = document.querySelector('.whatever');
const y = x.style;
```

### 🙁 Actual behavior

Given the two JavaScript (not TypeScript) lines above, `style` is detected as an error, with diagnostic `Property 'style' does not exist on type 'Element'.`.

On the other hand, pasting those two lines in chrome dev tools is just fine (provided there's an element with the `whatever` class).

### 🙂 Expected behavior

No error diagnosed, I guess.

**Comment by MartinJohns** [comment]
Duplicate of #16920. Used search terms: `style element in:title`

**Comment by Aster89** [comment]
Can somebody tell me how do I apply what is in #16920 (which seems to refer to TypeScript) to my usecase (JavaScript)?
